enclave/core/time.c

```diff
@@ -1,9 +1,12 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#include "openenclave/bits/time.h"
 #include <openenclave/bits/types.h>
 #include <openenclave/internal/calls.h>
+#include <openenclave/internal/raise.h>
 #include <openenclave/internal/time.h>
+#include "openenclave/bits/result.h"
```

Comment:
why is this one out of alphabetical order?
Also lines 4 and 9 use quotes whereas the other lines use <> which seems inconsistent